src/seals/diagnostics/__init__.py

```diff
@@ -7,3 +7,15 @@
     entry_point="seals.diagnostics.risky_path:RiskyPathEnv",
     max_episode_steps=5,
 )
+
+gym.register(
```

Comment:
I find these names confusing. Can we go with `InitShiftTrain` and `InitShiftTest` perhaps? I'm not actually sure which is which -- I think `InitShiftExpert` is the one the reward/imitation policy is learned in, and `InitShiftLearner` is the one it is tested in (i.e. policy deployed in or reward model optimized in)?